Example/Example/macOS/Code/TestStandardFormattersViewController.swift

```diff
@@ -0,0 +1,99 @@
+//
+//  TestStandardFormattersViewController.swift
+//  Example
+//
+//  Created by Artem Starosvetskiy on 20/11/2016.
+//  Copyright © 2016 Artem Starosvetskiy. All rights reserved.
+//
+
+import Foundation
+import Cocoa
+
+final class TestStandardFormattersViewController : TestFormattersViewController {
+
+}
+
+extension TestStandardFormattersViewController {
+
+    override func viewDidLoad() {
```

Comment:
Again, overriding methods in extensions doesn't seems good.
BTW, what the point of doing it in extension, while you have class declared a couple of lines above?